.gitignore

```diff
@@ -3,3 +3,6 @@ _site/
 node_modules/
 .DS_store
 .*.swp
+
+# Files specifically related to the compiled assets
+dist/
```

Comment:
Are we planning on pushing the dist/ folder somewhere so that these compiled files are available somewhere for non-dev folks?